Description: Add "foreign" option to automake so autoreconf works
 Avoid complaints about missing NEWS, README, etc.
Author: Roland Dreier <rbd@debian.org>
---
Index: libibmad-1.3.9/configure.in
===================================================================
--- libibmad-1.3.9.orig/configure.in	2012-04-30 13:42:43.000000000 -0700
+++ libibmad-1.3.9/configure.in	2012-11-27 12:05:31.000000000 -0800
@@ -6,7 +6,7 @@
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(config)
 AM_CONFIG_HEADER(config.h)
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign])
 
 AC_SUBST(RELEASE, ${RELEASE:-unknown})
 AC_SUBST(TARBALL, ${TARBALL:-${PACKAGE}-${VERSION}.tar.gz})
